libraries/engage/locations/components/StoreList/StoreListProductInfo.jsx

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React, { useContext } from 'react';
+import { Grid, ProductProperties } from '@shopgate/engage/components';
+import { ProductGridPrice, type ProductAware } from '../../../product';
+import { FulfillmentContext } from '../../locations.context';
+import { productInfo, productInfoLeft, priceInfo } from './StoreListProduct.style';
+
+/**
+ * Renders the product information of the store list.
+ * @returns {JSX}
+ */
+function StoreListProductInfo() {
+  const { product } : ProductAware = useContext(FulfillmentContext);
```

Comment:
You don't need to specify a type for the object. React's `useContext` will automatically recognise. AND, this seems to be the wrong type, since you are using the FulfillmentContext, which has the type of FulfillmentContextProps.